F4transkript/f4transkript.munki.recipe

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads the latest version of F5transkript and imports into Munki.</string>
+	<key>Identifier</key>
+	<string>com.github.its-unibas.munki.f4transkript</string>
+	<key>Input</key>
+	<dict>
+		<key>MUNKI_REPO_SUBDIR</key>
+		<string>apps/f4transkript</string>
+		<key>NAME</key>
+		<string>f4transkript</string>
+		<key>pkginfo</key>
```

Comment:
Add blocking applications to avoid installing the software when f4transkript is running.